azure-mgmt-graph-rbac/src/main/java/com/microsoft/azure/management/graphrbac/implementation/GraphRbacManager.java

```diff
@@ -136,21 +136,21 @@ public String tenantId() {
     /**
      * @return the Active Directory user management API entry point
      */
-    public Users users() {
-        if (users == null) {
-            users = new UsersImpl(this);
+    public ActiveDirectoryUsers users() {
+        if (activeDirectoryUsers == null) {
+            activeDirectoryUsers = new ActiveDirectoryUsersImpl(this);
         }
-        return users;
+        return activeDirectoryUsers;
     }
 
     /**
      * @return the Active Directory group management API entry point
      */
-    public Groups groups() {
-        if (groups == null) {
-            groups = new GroupsImpl(this);
+    public ActiveDirectoryGroups groups() {
```

Comment:
from what I can tell, this method was added in v1.1.0, so it'd need `@Beta(SinceVersion.V1_1_0)` on it.
Is there anything else like this?